src/Player/States/Run.gd

```diff
@@ -8,7 +8,7 @@ func unhandled_input(event: InputEvent) -> void:
 
 func physics_process(delta: float) -> void:
 	_parent.physics_process(delta)
-	if player.is_on_floor() or player.is_on_wall():
+	if player.is_on_floor() :
```

Comment:
Space left before the `:`. You can use a code formatter to address these things for you: https://github.com/Scony/godot-gdscript-toolkit/

Reply:
Thanks for the suggestion, I'll check it out.